src/Entities/Category.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+
+namespace RobotStores\Entities;
+
+
+class Category extends \RobotStores\Interfaces\CategoryInterface
```

Comment:
you dont extend an interface, you implement it. make this a real interface then implement it